wd_demo.au3

```diff
@@ -1145,6 +1146,10 @@ Func UserFile()
 	Local $aCmds = FileReadToArray($sScriptFileFullPath)
 	If @error Then Return SetError(@error, @extended)
 
+	Local Const $_VAR_SIZE = UBound($_VAR)
+	ReDim $_VAR[0] ; delete content
+	ReDim $_VAR[$_VAR_SIZE] ; reinitialize size
```

Comment:
Alternative for consideration that avoids multiple ReDims --

```autoit
	Local $aEmpty1D[UBound($_VAR)]
	$_VAR = $aEmpty1D
```

Reply:
Take a look: [CleanUp $_VAR](https://github.com/Danp2/au3WebDriver/pull/462/commits/e6fbed676c8baddba2de4428f02738cbf670a8ca)

Reply:
and:
[CleanUp again](https://github.com/Danp2/au3WebDriver/pull/462/commits/2f93c03d2b3b8af75f697714f04e0d10eea80284)

Reply:
> and: [CleanUp again](https://github.com/Danp2/au3WebDriver/pull/462/commits/2f93c03d2b3b8af75f697714f04e0d10eea80284)

How is changing `$aEmpty1D` to `$aCleanUP` an improvement? FWIW, I find the original version more intuitive / less confusing.

Reply:
Fix: [CleanUp Fix](https://github.com/Danp2/au3WebDriver/pull/462/commits/d9f109d0481cfba24a89abbb68c9334c528d9265)

Reply:
Ok 
Just revert it here [$aEmpty1D revert naming](https://github.com/Danp2/au3WebDriver/pull/462/commits/7253788e9605982a9117013886b4aa39888892b1)

And only check entire changes here:
https://github.com/Danp2/au3WebDriver/pull/462/files